docs/software/webserver_stack.md

```diff
@@ -1,25 +1,137 @@
+- [Tomcat8 - Apache Tomcat server](https://dietpi.com/phpbb/viewtopic.php?p=4316#p4316)  
+
+### Databases & Data stores
+
+- MariaDB
+- SQlite
+- [phpMyAdmin - Web interface SQL admin tool](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p54)  
+- Redis - A non-SQL based data store
+- InfluxDb
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
 
-!!! warning "Documentation in progress..."
+## Overview
+
+If you are you looking for an inexpensive way to operate your web servers or a dynamic websites, you could choose one of the web development stacks. They come with a complete package. Not only will you have the DietPi OS, web server, but you'll have the database software, and also the scripting language. 
+
+All stacks come with PHP caches (APCU and OPcache). The sizes of which are optimized based on a share of total memory.
+
+## [**LAMP Webserver - Apache2 / MariaDB / PHP**](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p52)
+
+LAMP stack is a popular open source web platform commonly used to run dynamic web sites and servers. It is considered by many, as the platform of choice for development and deployment of high performance web applications which require a solid and reliable foundation.
+
+![dietpi-webstack-lamp](../assets/images/dietpi-software-webstack-lamp.jpg)
+
+=== "Quick start"
+
+    **Access website:**
+
+    - url = http://192.168.0.100 (`http://<your.local.ip>` or `http://<your.host.name>`)
+    - local directory = /var/www
+
+    **Access phpinfo:**
+
+    - url = http://192.168.0.100/phpinfo.php
+
+    **Access memcache info:**
+
+    - APC/u = http://192.168.0.100/apc.php
+    - Opcache = http://192.168.0.100/opcache.php
+
+    **MySql / MariaDB Details:**
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+    - username = root
+    - password = dietpi
+
+=== "Secured access - HTTPS/SSL"
+
+    **Let's Encrypt** is highly recommended - [see here how to install](../../dietpi_tools/#lets-encrypt-ssl-nordvpn-support). This will automate the creation and setup of your free SSL cert.
+
+    ??? note "Alternative way: Manually enable HTTP/SSL by installing a self-signed SSL certificate"
+
+        !!! caution ""
+            This method is recommended only in case **Let's encrypt** is not a viable option. 
+    
+        **Step 1. Create key**
+
+        ``` bash
+        mkdir -p /etc/apache2/ssl
+        openssl req -x509 -nodes -days 1000 -newkey rsa:2048 -keyout /etc/apache2/ssl/apache.key -out /etc/apache2/ssl/apache.crt
+        ```
+
+        **Step 2. Enable SSL config and restart Apache**
+
+        ``` bash
+        chmod 600 /etc/apache2/ssl/*
+        cat << _EOF_ > /etc/apache2/sites-enabled/default-ssl.conf
```

Comment:
Please change `_EOF_` to `'_EOF_'` (single quotes), else below `${APACHE_LOG_DIR}` would be expended to an empty string in the resulting file and the logs are written to the root directory. No I know why I found this in some cases on DietPi systems.

Generally:
- Variable calls and command substitutions are expended inside "here documents", this way of creating content via `<< <EndWord>`.
- When something is copy&pasted and wanted literally in a file, the ending word needs to be single quoted.

I already fixed this in the forum.

Reply:
I will apply this change.